pkg/app/master/tui/main.go

```diff
@@ -0,0 +1,20 @@
+package tui
```

Comment:
Minor thing... i got confused initially when i saw main.go thinking that it has the `main` function and its own `main` package. Renaming the file to `tui.go` would eliminate the confusion.